pom.xml

```diff
@@ -28,11 +28,11 @@ under the License.
     <version>0.2-SNAPSHOT</version>
 
     <properties>
-        <flink.version>1.13.0</flink.version>
+        <flink.version>1.15.0</flink.version>
         <slf4j.version>1.7.15</slf4j.version>
         <log4j.version>1.2.17</log4j.version>
         <java.version>1.8</java.version>
-        <scala.binary.version>2.11</scala.binary.version>
```

Comment:
This change seems not related to this PR. What about split it into a separate PR?

Reply:
@dianfu, I have bumped the Flink version to 1.15.0 and this change is related to the bump.

Reply:
My point is why bump Flink version in this PR?

Reply:
OK. Thanks for the explanation.